app/models/ingredient.rb

```diff
@@ -0,0 +1,7 @@
+class Ingredient < ApplicationRecord
+  validates_presence_of :name
```

Comment:
Just as a reminder, you don't need to include validations in your ICs but it's great practice if you want to keep using them! The existing models will always have them but if you don't include them for any models you create, that's totally fine.